Homepwner/AppDelegate.swift

```diff
@@ -13,7 +13,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         // Assign a particular store versions to the ItemViewController
         if let itemsController = window?.rootViewController as? ItemsViewController {
-            itemsController.itemStore = RandomItemStore()
+            itemsController.itemStore = FixedItemStore() as ItemRepository
```

Comment:
It's not necessary to cast the FixedItemStore as an ItemRepository just as long as it conforms to the protocol in its class descriptor